specs/altair/p2p-interface.md

```diff
@@ -94,6 +112,92 @@ Unaggregated signatures (along with metadata) are sent as `SyncCommitteeSignatur
 
 Aggregated sync committee signatures are packaged into (signed) `SyncCommitteeContribution` along with proofs and gossiped to the `sync_committee_contribution_and_proof` topic.
 
+### Transitioning the gossip
+
+With any fork, the fork version, and thus the `ForkDigestValue`, change.
+Message types are unique per topic, and so for a smooth transition a node will have to subscribe to both the old and new topics.
+
+All remaining topics are updated. In advance of the fork, a node SHOULD subscribe to the post-fork variants of the topics.
+
+Subscriptions are expected to be well-received, all updated nodes should subscribe as well.
+Topic-meshes can be grafted quickly as the nodes are already connected and exchanging gossip control messages.
+
+Messages SHOULD NOT be re-broadcast from one fork to the other.
+
```

Comment:
```suggestion
A node's behavior before the fork and after the fork are as follows:
```

I think adding a bit of prose here to guide the reader will help. I was a bit confused when we just dropped into pre and post and just an additional sentence that this is the expected behavior in each era is helpful